web/routes/index.js

```diff
@@ -23,11 +23,11 @@ router.get('/', async (req, res) => {
   
   try {
     products = await getProducts();
-  }
-  catch (e) {
-    console.error('error: unable to fetch products');
-    console.debug(e);
-    products = [{name: '[DATABASE ERROR: unable to fetch products] ' + Date.now()}];
+  } catch (err) {
+  // products = await getProducts().catch(err => {
+    // console.error('error: unable to fetch products' + err);
```

Comment:
when running npm test, this error will come up since it's not connect to db in container.  Is that fine?

Reply:
I think it's fine for this milestone. Can you open an issue to suppress it for v0.1.1? Can we provide a database mock so that it actually "connects" to "something" as part of the test?